test/gtest/common/googletest/gtest-param-test.h

```diff
@@ -457,6 +457,11 @@ internal::CartesianProductHolder<Generator...> Combine(const Generator&... g) {
 #define GTEST_GET_FIRST_(first, ...) first
 #define GTEST_GET_SECOND_(first, second, ...) second
 
+// Modified to reduce stack frame size per test file.
```

Comment:
is there a way to remove changes to googletest (disabling check for it)? or make it outside of googletest by including `gtest-param-test.h`?

Reply:
Outdated, done in other PRs